auth-foundation/src/main/java/com/okta/authfoundation/client/internal/NetworkUtils.kt

```diff
@@ -117,6 +122,47 @@ internal suspend fun <T> OidcConfiguration.internalPerformRequest(
     }
 }
 
+internal suspend fun OidcConfiguration.internalGetResponseWithRateLimitHandlingFor(
+    request: Request
+): Response {
+    currentCoroutineContext().ensureActive()
+    val rateLimitErrorCode = 429
+
+     return withContext(ioDispatcher) {
```

Comment:
You shouldn't need the `withContext`. All threading should be handled elsewhere already (I think).